client/forest-manager/src/tests.rs

```diff
@@ -0,0 +1,68 @@
+use common::types::HashT;
```

Comment:
Maybe we should rename it to `test_utils.rs` or something like that? It's not too intuitive to have a file called `tests.rs` that actually has no tests.